pgjdbc/src/test/java/org/postgresql/test/ssl/LazyKeyManagerTest.java

```diff
@@ -21,6 +23,16 @@
 
 public class LazyKeyManagerTest {
 
+  @Test
+  public void testLoadP12Key() throws Exception {
+    String certdir = "../certdir/";
+    PKCS12KeyManager pkcs12KeyManager = new PKCS12KeyManager(certdir + "goodclient.p12", new TestCallbackHandler("sslpwd"));
```

Comment:
I guess it is better to take `sslpwd` from the properties file.
Otherwise, the failures would be hard to analyze.

Reply:
agreed